<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>553</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>553</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>553</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>0</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>0</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>0</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>128</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>128</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>128</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>128</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>256</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>256</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>256</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>256</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>384</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>384</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>384</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>384</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRestart">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHowTo"/>
   </widget>
   <widget class="QMenu" name="menuAI">
    <property name="title">
     <string>AI</string>
    </property>
    <addaction name="actionAuto"/>
    <addaction name="actionDelayBetweenMove"/>
    <addaction name="actionTrain"/>
    <addaction name="actionSearchDepth"/>
    <addaction name="actionLearnRate"/>
   </widget>
   <addaction name="menuRestart"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAI"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionHowTo">
   <property name="text">
    <string>HowTo</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionDelayBetweenMove">
   <property name="text">
    <string>DelayBetweenMove</string>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>Train</string>
   </property>
  </action>
  <action name="actionSearchDepth">
   <property name="text">
    <string>SearchDepth</string>
   </property>
  </action>
  <action name="actionLearnRate">
   <property name="text">
    <string>LearnRate</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
